github-dark.css

```diff
@@ -2787,18 +2788,16 @@
   .blog .text-delta, .blog .text-gamma, .blog h1, .blog h2, .blog h3, .blog h4,
   .blog h5, .site-header-nav a, .menu-item.selected, .overview .text-beta,
   article h2, #message-list h1, #message-list h3, .graph-page h1, .graph-page h3,
-  .dashboard-notice h2,
   .prose-diff.collapsed .rich-diff-level-zero.expandable:hover .octicon,
-  .counter, .Counter, .state, .State, a.state, a.State,
-  .site-header-dark .site-header .site-header-link,
+  .counter, .Counter, .state, .State, a.state, a.State, .dashboard-notice h2,
+  .site-header-dark .site-header .site-header-link, .version-picker a:hover,
   .markdown-format .octicon-link, .Box-row--hover-blue.link-gray:hover,
```

Comment:
Please add all new selectors to the end of the block, next to the `{`.

Reply:
They are not new entries they were moved within the rule to cleanup.

Reply:
Yes, it makes the css pretty, but consistently adding new/modified selectors to the end helps us estimate the age of a selector. The class names in older selectors may not even exist anymore, then we can decide to remove them without much debate.

Reply:
OK. Ill know for next time.